jetty-server/src/main/java/org/eclipse/jetty/server/handler/IPAccessHandler.java

```diff
@@ -358,7 +404,10 @@ public String dump()
         buf.append(toString());
         buf.append(" BLACKLIST:\n");
         dump(buf, _black);
-
+        buf.append(" CONNECTOR NAMES:\n");
+        for (String connectorName : _connectorNames) {
```

Comment:
I would expect this to be simply `dump(buf, _connectorNames);`

Reply:
yeah i was confused here. @gregw any thoughts?

Reply:
whoa!  This dump is really bad/old/wrong to start with!!  See dump style in InetAccessHandler.

Reply:
@joakime i tried that suggestion and it is a compile error. I can move the dump implementation stuff here instead